samples/DataGridListSample.js

```diff
@@ -67,7 +67,8 @@ enyo.kind({
 				selected: false,
 				text: "Item " + idx + title,
 				subText: subTitle,
-				url: "http://placehold.it/300x300/" + Math.floor(Math.random()*0x1000000).toString(16) + "/ffffff&text=Image " + idx
+				url: "http://placehold.it/300x300/" + Math.floor(Math.random()*0x1000000).toString(16) + "/ffffff&text=Image " + idx,
+				placeholder: enyo.Image.placeholder
```

Comment:
I think we can probably remove this as the default value of `placeholder` is `enyo.Image.placeholder`